django/db/models/expressions.py

```diff
@@ -477,6 +477,35 @@ def desc(self, **kwargs):
         return OrderBy(self, descending=True, **kwargs)
 
 
+class ResolvedOuterRef(F):
+    """
+    An object that contains a reference to an outer query.
+
+    In this case, the reference to the outer query has been resolved, because the
+    inner query has been used as a subquery.
+    """
+    def as_sql(self, *args, **kwargs):
+        raise ValueError(
```

Comment:
In cases like this, put the closing ) on the next line:
```
raise ValueError(
    'This queryset contains a reference to an outer query and may '
    'only be used in a subquery.'
)
```
(and while longer lines are acceptable, wrap strings like this).